kratos/solving_strategies/convergencecriterias/convergence_criteria.h

```diff
@@ -77,18 +59,23 @@ class ConvergenceCriteria
     ///@name Type Definitions
     ///@{
 
+    /// Pointer definition of ConvergenceCriteria
+    KRATOS_CLASS_POINTER_DEFINITION(ConvergenceCriteria);
+    
+    /// Data type definition
     typedef typename TSparseSpace::DataType TDataType;
+    /// Matrix type definition
```

Comment:
I feel that this comment is not necessary .. !! The next line is already self explaining what it is doing !!

Reply:
Yeah, but if some day the Doxygen is used it will not create the documentation from nothing